src/Components/BookCardContainer/BookCardContainer.tsx

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
 import BookCard from '../BookCard/BookCard';
+import Error from '../Error/Error'
 import './BookCardContainer.css';
+import NavBar from '../NavBar/NavBar'
 
 interface Props {
   allBooks: { title: string, cover_id: number, key: any, id: string, authors: [] }[],
-  oneBook: Function
+  oneBook: Function,
+  error: boolean
 }
 
-const BookCardContainer: React.FC<Props> = ({ allBooks, oneBook }) => {
+const BookCardContainer: React.FC<Props> = ({ allBooks, oneBook, error }) => {
 
   const bookCards = allBooks.map((book) => {
     return (
-      <BookCard title={book.title} cover_id={book.cover_id} key={book.key} id={book.key} oneBook={oneBook} authors={book.authors} />
+     <BookCard title={book.title} cover_id={book.cover_id} key={book.key} id={book.key} oneBook={oneBook} authors={book.authors} />
     )
   })
 
   return (
     <div className="book-card-container">
-      {bookCards}
+      {error ? <Error /> : bookCards}
```

Comment:
Nice conditional rendering here! I know Raquel had another idea for error handling we can talk about just to see how it's done a few different ways.